manager.spec.yaml

```diff
@@ -1093,6 +1094,24 @@ components:
         maxItems: 2
       example: true,false
 
+    affecting_host_type:
+      in: query
+      name: affecting_host_type
+      description: >-
+        Controls, whenever CVE has 1 or more affecting systems. Value "edge" returns CVEs with one or more
+        vulnerable edge systems, value "rpmdnf" returns CVEs with one or more vulnerable rpmdnf systems. Value "none"
```

Comment:
Let's use 
```
s/edge systems/immutable systems/
```
and
```
s/rpmdnf systems/conventional systems/
```
to be consistent with UI and public terminology. Value `"edge"` can stay as is.

Reply:
good point